electron/index.js

```diff
@@ -74,6 +74,41 @@ function createWindow () {
     callback({ requestHeaders: details.requestHeaders });
   });
 
+  // Do not accept all kind of web permissions (camera, location...)
+  // https://electronjs.org/docs/tutorial/security#4-handle-session-permission-requests-from-remote-content
+  session.defaultSession
+    .setPermissionRequestHandler((webContents, permission, callback) => {
+      if (!url.startsWith('file:')) {
```

Comment:
`url` refers to the library here. You forgot: `const url = webContents.getURL()`

Reply:
F*ck... Nice catch, thanks!